run-nitro-enclaves-acm-tests

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+ERR_MISSING_BIN=1
+ERR_MISSING_FILE=2
+
+ERR_CODE=0
+
+# Binaries which should exist under $PATH after package installation
+acm_ne_bins_array=("nitro-cli" \
+		   "vsock-proxy" \
+		   "p11ne-agent" \
+		   "p11ne-client" \
+		  )
+
+# Files brought in by the RPM(s), specified by full path
+acm_ne_files_array=("/lib/systemd/system/nitro-enclaves-allocator.service" \
```

Comment:
/lib/systemd doesn't exist on all Linux distros. Do we care that we will only be able to run this script on distros with systemd?

Reply:
Indeed if this was a script to be used by the common user, this would have been a problem. However this script is intended to be included in a separate test RPM, which will only be used on an AL2 instance for testing purposes.

Reply:
Makes sense